conf/routes

```diff
@@ -49,6 +49,7 @@ GET           /streams/:stream_id/outputs
 GET           /streams/:stream_id/alerts                                                         @controllers.AlertsController.index(stream_id: String)
 POST          /streams/:stream_id/alerts/conditions/add/messagecount                             @controllers.AlertsController.addTypeMessageCount(stream_id: String)
 POST          /streams/:stream_id/alerts/conditions/add/fieldvalue                               @controllers.AlertsController.addTypeFieldValue(stream_id: String)
+POST          /streams/:stream_id/alerts/conditions/add/fieldstringvalue                         @controllers.AlertsController.addTypeFieldStringValue(stream_id: String)
```

Comment:
The path should be named the same as the condition: `/streams/:stream_id/alerts/conditions/add/fieldcontentvalue`